src/test/mon/test-mon-msg.cc

```diff
@@ -224,8 +224,8 @@ class MonMsgTest : public MonClientHelper,
   Cond cond;
 
   MonMsgTest() :
-    reply_msg(NULL),
     MonClientHelper(g_ceph_context),
+    reply_msg(NULL),
     lock("lock") { }
 
```

Comment:
I thought about the same, but I didn't give more importance as reply_msg was already used in the member initializer list. http://en.cppreference.com/w/cpp/language/initializer_list.

Fixed as per your review comment. Please verify now. Thanks.